src/components/Community/Meet/index.js

```diff
@@ -110,7 +111,24 @@ CommunityIssue.propTypes = {
   color: PropTypes.string
 }
 
-export default function CommunityMeet({ issues, theme, topics }) {
+export default function CommunityMeet({ theme }) {
+  const [isIssuesLoaded, setIsIssuesLoaded] = useState(false)
+  const [issues, setIssues] = useState([])
+
+  const [isTopicsLoaded, setIsTopicsLoaded] = useState(false)
+  const [topics, setTopics] = useState([])
```

Comment:
nit: perhaps we could use a custom hook for these API calls, to keep things consistent:

```js
const useAPICall = (apiFn, ...args) => {
  const [loading, setLoading] = useState(false)
  const [result, setResult] = useState(null)

  useEffect(() => {
    setLoading(true)
    setResult(null)
    apiFn(...args).then(result => {
      setLoading(false)
      setResult(result)
    })
  }, [apiFn, ...args])

  return { loading, result }
}

// ...
const topics = useAPICall(getLatestIssues, /* args for API call can go here */)

topics.loading // bool
topics.result // any
```

Reply:
Good idea, I'd probably would add error state here too.